lib/index.d.ts

```diff
@@ -754,7 +754,8 @@ declare namespace Stardog {
             database: string,
             query: string,
             /** Defaults to false. */
-            shared: boolean
+            shared: boolean,
+            reasoning?: boolean,
```

Comment:
Can you add the `description` type here too with a comment explaining why it's optional? It would also help as separation to establish that `reasoning` does not also fall into the bucket of `Defaults to false.`

Additionally, you'll want to add these to the body description for doc generation [here](https://github.com/stardog-union/stardog.js/pull/196/files#diff-83b6ae62f66182e10e7cca8617f11ac0R10) and run `yarn run docs` to update the README too.